### DISCUSSION TITLE
Storipress integration

### DISCUSSION THREAD

**Discussion by curtissrickard** [webstudio-is/webstudio-community#81] [Wishlist]
An integration between webstudio.is and storipress.com is the perfect match that would revolutionize the way you create and publish content.

By integrating webstudio.is with storipress.com you'd unleash the power of collaboration, empowering users to collaborate seamlessly and efficiently. Imagine the convenience of designing and building your website with webstudio.is while harnessing the content creation and optimization capabilities of Storipress.

With this integration, you can confidently streamline your entire content workflow, from ideation to publication, without any hassle or friction.

By joining forces, webstudio.is and storipress.com would create a powerhouse solution that boosts productivity, enhances collaboration, and accelerates content creation like never before.

I would be complete 🤍🥳

**Comment by kof** [comment]
Storipress was just recently mentioned here https://github.com/webstudio-is/webstudio-community/discussions/47#discussioncomment-7738770 I am starting to wonder what's all the hype is about? One big downside is its not open source.

**Comment by kof** [comment]
closing in favor of https://github.com/webstudio-is/webstudio-community/discussions/47
